solutions/devsprint-ana-nogal-1/DeliveryApp/Screens/Home/HomeViewController.swift

```diff
@@ -7,7 +7,11 @@
 
 import UIKit
 
-class HomeViewController: UIViewController {
+class HomeViewController: UIViewController, UISearchResultsUpdating {
```

Comment:
```suggestion
class HomeViewController: UIViewController {
```
O que vc acha de criar antes uma extenção:
```
extension HomeViewController: UISearchResultsUpdating {}
```
Vc precisa de conformar com esse protocol? Ou com o `UISearchBarDelegate`

Reply:
corrigido.